pkg/storage/ingest/config.go

```diff
@@ -76,6 +76,9 @@ type KafkaConfig struct {
 
 	AutoCreateTopicEnabled           bool `yaml:"auto_create_topic_enabled"`
 	AutoCreateTopicDefaultPartitions int  `yaml:"auto_create_topic_default_partitions"`
+
+	// Used when logging unsampled client errors. Set from ingester's ErrorSampleRate.
+	ClientErrorSampleRate int64 `yaml:"-"`
```

Comment:
[nit] Have you considered prefixing it with "Fallback"?

Reply:
It looked too long to me, but since you're asking about it too, I've added it.